src/App.js

```diff
@@ -16,31 +80,45 @@ const genres = [
 ]
 
 function App() {
-    const [selected, setSelected] = useState(genres[0].name);
+    const [selectedGenre, setSelectedGenre] = useState(genres[0].name);
+	const [selectedMovie, setSelectedMovie] = useState();
+	const [sortedBy, setSortedBy] = useState();
 
-	const onSelect = (event) => {
-		event.preventDefault()
-		setSelected(event.target.dataset.name)
+	const sortMovies = () => {
```

Comment:
this name is not descriptive. this function returns sorted movies, not sorting them.